backend/src/controllers/adminController.ts

```
@@ -0,0 +1,409 @@
+        success: false,
+        error: 'Admin access required',
+      });
+      return;
+    }
+
+    const { page = '1', limit = '10', status, search } = req.query;
+    const pageNum = parseInt(page as string);
+    const limitNum = parseInt(limit as string);
+    const skip = (pageNum - 1) * limitNum;
+
+    let where: any = {};
```

ESLint

[error] 'where' is never reassigned. Use 'const' instead.

(prefer-const)